### PR TITLE
Feature/ps 10 setup database

### PR DESCRIPTION
**Pull request by MatasLa**
Initital SQL/Entity framework set up for the project.
Parcel model is currently a mockup.

### CODE REVIEW

File: Infrastructure/DatabaseContext.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure
+{
+    public class DatabaseContext : DbContext
+    {
+        public DatabaseContext(DbContextOptions<DatabaseContext> options) : base (options) { }
```
Comment (Stokalas) [inline Infrastructure/DatabaseContext.cs:13]: Build warning (check Github actions). Not the end of the world, but I think we should avoid these at least in such early stage of project.

File: Infrastructure/Models/Parcel.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Models
+{
+    public class Parcel
+    {
+        public int Id { get; set; }
+        [Key]
+        public string TrackingNumber { get; set; }
```
Comment (Stokalas) [inline Infrastructure/Models/Parcel.cs]: Same here with warning

File: Infrastructure/Models/Parcel.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Models
+{
+    public class Parcel
```
Comment (Stokalas) [inline Infrastructure/Models/Parcel.cs:10]: If this just for mock purposes and full Model (more fields) will be implemented in future then OK.

File: WebAPI/appsettings.json [hunk 4]
```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=localhost\\SQLEXPRESS;Database=ParcelDB;Trusted_Connection=True;"
```
Comment (Stokalas) [inline WebAPI/appsettings.json]: Maybe adding 'Encrypt=True' would be nice :)

**Review by Stokalas — CHANGES_REQUESTED** [review]
Overall well done, main thing to fix is the warnings, other comments are just suggestions.

File: Infrastructure/Models/Parcel.cs (hunk 3 above)
Comment (MatasLa) [reply Infrastructure/Models/Parcel.cs:10]: Currently it is just for mock purposes.

**Comment by MatasLa** [comment]
Fixed warnings, added encrypt.

**Review by Stokalas — APPROVED** [review]

**Pull request merged**